images/linux/toolsets/toolset-2004.json

```diff
@@ -106,6 +106,7 @@
     "apt": {
         "common_packages": [
             "build-essential",
+            "chromium-browser",
```

Comment:
I suggest moving it to cmd_packages because those are packages with executables

Reply:
We've decided to leave the separate test for chromium browser.